_data/Tinggaard/Tinggaard.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Jens Tinggaard # No longer than 28 characters
+institution: Odense Tekniske Gymasium 🚩 # no longer than 58 characters
```

Comment:
Hey @Tinggaard,
Could you please remove this flag as it might not be recognized? - Apart from this, everything else looks good to me.

Reply:
@Abhishekds94 Thanks for reviewing! I think it's fine. Based on [this](https://github.com/education/graduation/issues/348#issuecomment-631776551) comment, it's mentioned they can do emojis too!